config/default.config

```diff
@@ -13,7 +13,7 @@ params {
     dataset_id = ''
     blcds_registered_dataset = false
 
-    ucla_cds = true
+    ucla_cds = false
```

Comment:
**question (non-blocking)**: Is this set to `false` since the pipeline will be used primarily outside of our cluster?

Reply:
The goal is for this to be broadly used so I'm trying to set convenient defaults. Is it more problematic for UCLA users to forget to set this to `true` or for non-UCLA users to forget to set this to `false`?

Reply:
It's not super problematic since this pipeline isn't really too disk I/O heavy